plugins/lua/hotkeys.lua

```diff
@@ -41,6 +40,34 @@ function HotspotMenuWidget:overlay_trigger()
         mouseover=self.mouseover}:show()
 end
 
+local dscreen = dfhack.screen
+
+function HotspotMenuWidget:onRenderBody(dc)
+    local tpos = dfhack.textures.getDfhackLogoTexposStart()
+    local x, y = dc.x, dc.y
+
+    if tpos == -1 then
```

Comment:
Should this also be checking for graphics mode? e.g. if the texture was previously loaded but DF is now in classic mode

Reply:
Yes, but my point is I only see one being painted here at a time, and it looks to me like it's switched based on whether the texture was ever loaded, not whether the texture can be displayed right now. But I could be misunderstanding.

Reply:
The first branch paints just the characters if the texture is unavailable since otherwise the `tpos+N` lines would refer to incorrect textures. The second branch paints both, suitable for both graphics and non-graphics modes. Notice that both `ch` *and* `tile` are populated in the second branch.

Reply:
Ohh, I did notice that a few days ago but forgot to ask about it when actually reviewing. Thanks!